site/content/integrate/apps/testing/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Testing"
+heading: "Apps testing"
+description: "How to test the App framework"
+weight: 81
+---
+
+This page describes how to test changes made to the App Framework.
+
+## E2E Testing
+
+To run end-to-end testing with Cypress you need to do the following:
+
+1. Clone and enter the [`mattermost-app-examples`](https://github.com/mattermost/mattermost-app-examples) repository:
+
+```bash
+git clone https://github.com/mattermost/mattermost-app-examples
+cd mattermost-app-examples
+```
+
+2. Launch the provided Mattermost server
+
+```bash
+docker compose up
```

Comment:
A little unfortunate that the e2e tests rely on the separate repo. Maybe have a small excerpt explaining why we need to clone this repo? Something like

> We will use this project's Docker Compose configuration to run our test server.

Reply:
it's true, I'm not sure if it's the case or not, but, I'll continue investigating, thank you!